packages/epo-react-lib/src/layout/BasicModal/BasicModal.tsx

```diff
@@ -0,0 +1,43 @@
+import IconComposer from "@/svg/IconComposer";
+import * as Styled from "./styles";
+import { FunctionComponent, MouseEvent, ReactNode } from "react";
+
+interface BasicModalProps {
+  children: ReactNode;
+  open?: boolean;
+  onClose: (event?: MouseEvent) => void;
+  darkMode?: boolean;
+  title?: string;
+  description?: string;
+}
+
+const BasicModal: FunctionComponent<BasicModalProps> = ({
```

Comment:
BasicModal is an adaptation of rubin-obs-client's BasicModal and AuthModal, it's not exactly either it's closer to being both. There are optional title and description content insertions but otherwise it wraps whatever children it is given.